packages/flutter/lib/src/material/text_field.dart

```diff
@@ -581,6 +582,13 @@ class TextField extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.cursorRadius}
   final Radius? cursorRadius;
 
+  /// Whether the cursor will animate from fully transparent to fully opaque
+  /// during each cursor blink.
+  ///
+  /// By default, the cursor opacity will animate on iOS platforms and will not
+  /// animate on Android platforms.
```

Comment:
Can you use a macro/template here instead of duplicating this?  I see some discussion above about a macro not working before but let me know if you can't get it.